ARViewer/ViewController.swift

```diff
@@ -116,6 +119,19 @@ class ViewController: UIViewController, ARSCNViewDelegate, SCNPhysicsContactDele
     // MARK: - Actions
     
     @IBAction func didTapScreen(_ sender: UITapGestureRecognizer) { // fire bullet in direction camera is facing
+        
+        do
```

Comment:
this should be modularized into a function since it remains almost identical three times

you can use an enum to distinguish between the audio files